nix/lib.nix

```diff
@@ -15,6 +15,11 @@
       ];
       serviceModule = { config, name, ... }: {
         options = {
+          dataDir = lib.mkOption {
+            type = lib.types.str;
+            default = "./data/${name}";
+            description = "The ${service}.<name> data directory";
```

Comment:
@srid How does this look for a description? In the docs the option will be rendered as `<name>` anyways?

context: https://github.com/juspay/services-flake/pull/258#discussion_r1666999402

Reply:
Sure (but hard to say definitely without rendering the docs though)